### ISSUE TITLE
React type export doesn't include `title`

### ISSUE THREAD

**Issue by jharwig** [tailwindlabs/heroicons#810]
Since https://github.com/tailwindlabs/heroicons/pull/779, the `title` prop can be set to create an svg title element, but the react typings do not include that prop: since the the definition in the Icon.d.ts file doesn't include it:

```ts
declare function StarIcon(props: React.ComponentProps<'svg'>): JSX.Element;
```

The error message is `Property 'title' does not exist on type 'IntrinsicAttributes & SVGProps<SVGSVGElement>'.`

I think the vue types are working because it allows any [`HTMLAttributes` through](https://github.com/tailwindlabs/heroicons/blob/4658f973a30bb2791b94a306dda2505003bbdbf9/scripts/build.js#L97), so it's probably just a matter of adding a `& {title?: string }` to the react props...

```diff
-? `import * as React from 'react';\ndeclare function ${componentName}(props: React.ComponentProps<'svg'>): JSX.Element;\nexport default ${componentName};\n`
+? `import * as React from 'react';\ndeclare function ${componentName}(props: React.ComponentProps<'svg'> & {title?: string}): JSX.Element;\nexport default ${componentName};\n`
```

**Comment by RobinMalfait** [comment]
Hey! Thank you for your PR!
Much appreciated! 🙏

This should be fixed by #814, and will be available in the next release.

You can already try it using:
- `npm install @heroicons/react@insiders`.

**Comment by kunklejr** [comment]
Looks like there have been 3 releases with this update already!
